sechub-shared-kernel/src/main/java/com/daimler/sechub/sharedkernel/mapping/MappingIdentifier.java

```diff
@@ -12,16 +12,17 @@
 
 /**
  * Represents all mapping identifiers.
+ * 
  * @author Albert Tregnaghi
  *
  */
 @MustBeKeptStable("The enum is used as identifiers in database + in code. Do NOT rename Ids inside")
 public enum MappingIdentifier {
-   
-    CHECKMARX_NEWPROJECT_TEAM_ID("checkmarx.newproject.teamid", MappingType.ADAPTER_CONFIGURATION),
 
-    CHECKMARX_NEWPROJECT_PRESET_ID("checkmarx.newproject.presetid",MappingType.ADAPTER_CONFIGURATION),
-    
+    CHECKMARX_NEWPROJECT_TEAM_ID("checkmarx.newproject.teamid.mapping", MappingType.PRODUCT_EXEC_CFG_PARAM),
```

Comment:
Please change the name from `PRODUCT_EXEC_CFG_PARAM` to `PRODUCT_EXEC_CONFIG_PARAM`. `CFG` has many meanings in contrast to `CONFIG` which is very clear. Saving, three letters is not worth the trouble having difficulties understanding the abbreviation `CFG`.

CFG for me means Control flow graph or context-free grammar. Wikipedia list another abbreviation related to computer science: Configuration file (https://en.wikipedia.org/wiki/CFG).